SimulationAgent/SimulationRunner.cs

```diff
@@ -230,6 +231,13 @@ public void Stop()
             }
         }
 
+        public int GetActiveDeviceCount()
+        {
+            int activeDeviceCount = this.deviceStateActors.Count(a => a.Value.IsDeviceActive);
+            
+            return activeDeviceCount;
```

Comment:
merge the 2 lines, and return directly (like the change to `DeviceStateActor.cs` above)